qa/.gitignore

```diff
@@ -2,3 +2,4 @@
 .*.sw[nmop]
 *.pyc
 .tox
+.mypy_cache
```

Comment:
remove this hunk. See #33510

Reply:
@toabctl could you address this comment?

Reply:
Done